easybuild/easyconfigs/p/picard/picard-2.18.14-Java-1.8.eb

```
@@ -0,0 +1,48 @@
+##
+# This is an easyconfig file for EasyBuild, see https://github.com/easybuilders/easybuild
+#
+# This is a contribution from Phoenix HPC Service, The University of Adelaide, Australia
+# Homepage: https://www.adelaide.edu.au/phoenix/
+#
+# Copyright::   adelaide.edu.au/phoenix
+# Authors::	Robert Qiao <robert.qiao@adelaide.edu.au>, Exe Escobedo <exequiel.sepulvedaescobedo@adelaide.edu.au>
+# License::	MIT 
+#
+# 2.10.1:
+# Adam Huffman
+# The Francis Crick Institute
+# 2.18.11:
+# Jonas Demeulemeester
+# The Francis Crick Institute
+##
+
+easyblock = 'JAR'
+
+name = 'picard'
+version = '2.18.14'
+versionsuffix = '-Java-%(javaver)s'
+
+homepage = 'http://sourceforge.net/projects/picard'
+description = """A set of tools (in Java) for working with next generation sequencing data in the BAM format."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
```

@zao Is just noticed that the recent existing `picard` have this too, but it's wrong, the toolchain version should *not* be `dummy`, since that implies not loading the `Java` dependency during the installation.

It's not a huge issue, since the `JAR` easyblock just copies the `.jar` file, but it's good to keep this consistent...

```suggestion
toolchain = {'name': 'dummy', 'version': ''}
```

Fix for recent `picard` easyconfigs in #7116.

---

Ah, didn't know that there was a difference in semantics there. Suggestion applied.

---

Yeah, it's something we want to get rid of (cfr. https://github.com/easybuilders/easybuild-framework/pull/1233), it's a horrible historical mistake...